firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalDocumentsView.java

```diff
@@ -142,7 +142,16 @@ Document getDocument(DocumentKey key) {
     return computeViews(docs, overlays, new HashSet<>());
   }
 
-  /*Computes the local view for doc */
+  /**
+   * Computes the local view for the given documents.
+   *
+   * @param docs The documents to compute views for. It also has the base version of the documents.
+   * @param overlays The overlays that need to be applied to the given base version of the
+   *     documents.
+   * @param existenceStateChanged A set of documents that might there existence states changed. This
```

Comment:
s/that might there existence states changed/whose existence states might have changed.